arch/arm/boot/dts/meson8m2-minix-neo-x8-h-plus.dts

```diff
@@ -0,0 +1,98 @@
+// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
+/*
+ * Copyright (c) 2018 Oleg Ivanov <balbes-150@yandex.ru>
+ * Copyright (c) 2018 Martin Blumenstingl <martin.blumenstingl@googlemail.com>
+ * Copyright (c) 2020 Arnaud Mondit <arnaud.mondit@gmail.com>
```

Comment:
For this I prefer a copyright style similar to what we have in ``arch/arm64/boot/dts/amlogic/meson-gxl-s905w-tx3-mini.dts`` to make it clear that not all of us worked on that file

Reply:
Fixed that on the last commit, I hope it's better?